views/success.pug

```diff
@@ -0,0 +1,24 @@
+doctype html
+head
+  meta(charset='UTF-8')
+  meta(name='viewport' content='width=device-width, initial-scale=1.0')
+  link(rel='stylesheet' href='https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css' integrity='sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh' crossorigin='anonymous')
+  link(rel='stylesheet' href='cart.css')
+  title Confirmation
+nav.navbar.navbar-expand-lg.navbar-light.bg-light
+  .container
+    a.navbar-brand(href='/') Store
+
+section
+  .container
+    .row
+      .col-lg-4.col-md-6
+        h1 Thank you, we have received your order
```

Comment:
Would it be possible to get the order total and the shipping address on this page as well?

Reply:
Yes it is.